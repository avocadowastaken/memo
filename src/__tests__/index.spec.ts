import * as api from "../index";

it("exposes public api", () => {
  expect(api).toMatchSnapshot();
});
