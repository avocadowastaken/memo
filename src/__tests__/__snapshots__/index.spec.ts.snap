// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exposes public api 1`] = `
Object {
  "MemoCache": [Function],
  "createMemo": [Function],
  "createPromiseMemo": [Function],
}
`;
